cli/proc_state.rs

```diff
@@ -258,6 +267,7 @@ impl ProcState {
       lockfile.as_ref().cloned(),
     )
     .await?;
+    npm_resolver.add_package_reqs(package_json_reqs).await?;
```

Comment:
This unconditionally downloads all "dependencies" and "devDependencies" from package.json. I think we should be smarter about the latter, but maybe that's fine for the first pass.